Constraint/StudentApplicationConstraint.sql

```diff
@@ -1,5 +1,5 @@
 ALTER TABLE
     [dbo].[StudentApplication]
 ADD
-    CONSTRAINT ForeignKeyStudent FOREIGN KEY (StudentID) REFERENCES [dbo].[Student](StudentID),
-    CONSTRAINT ForeignKeyDocument FOREIGN KEY (DocumentID) REFERENCES [dbo].[Document](DocumentID),
\ No newline at end of file
+    CONSTRAINT ForeignKey_University FOREIGN KEY (UniversityID) REFERENCES [dbo].[UniversityID](UniversityID),
```

Comment:
could you please write this in CamelCase `ForeignKey_University` for consistency with the others and the rest of the project,